WORKSPACE

```diff
@@ -61,6 +38,15 @@ load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_depen
 
 buildifier_dependencies()
 
+# rules_cc
+
+http_archive(
+    name = "rules_cc",
```

Comment:
Where do we use `rules_cc`?

Reply:
It's a dependency of one of our dependencies. Without specifying a more up to date version, something in one of our dependencies' starlark functions breaks.